doc/authoring_help.md

```diff
@@ -61,7 +61,7 @@ examples:
 
 You can also document groups using the same format.
 
-```py
+```python
```

Comment:
I see "Python" in other places in this PR. Which one is correct?

Reply:
@qwordy 

Per the contributors guide, all code-blocks should be lower case, though Docs renders it capitalized. I've corrected the capitalized versions.